lib/pdk/cli/exec.rb

```diff
@@ -46,6 +46,13 @@ def self.git(*args)
         git_path = ENV['PDK_USE_SYSTEM_BINARIES'].nil? ? File.join(git_bindir, 'git') : 'git'
         execute(git_path, *args)
       end
+
+      def self.bundle(*args)
+        bundle_path = ENV['PDK_USE_SYSTEM_BINARIES'].nil? ? File.join(pdk_basedir, 'private', 'ruby', '2.1.9', 'bin', 'bundle') : 'bundle'
```

Comment:
This conflicts with https://github.com/puppetlabs/pdk/pull/58/commits/2d0ca925a57f0db56da62674191f6f348f6dfb75 from #58 - maybe I should pull this change out to get it merged independently, so we can leave the `PDK_USE_SYSTEM_BINARIES` behind us.